Bottom: fbe7c300573216a2a0e8e4ebd58c74c21e9372e8
Top:    c014a04bcdcfa18d83108b109eb9100d4df46ff6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-06-27 14:20:14 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index c5159ba..da2e3b3 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2752,7 +2752,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
